DataStructures/Libraries/src/Graph.java

```diff
@@ -0,0 +1,81 @@
+public class Graph
```

Comment:
Comments are missing in some files, Add suitable comments in all the java files.